godot-bindings/src/lib.rs

```diff
@@ -70,6 +92,20 @@ mod prebuilt {
             .unwrap_or_else(|e| panic!("failed to write gdextension_interface.rs: {e}"));
         watch.record("write_header_rs");
     }
+
+    pub(crate) fn get_godot_version() -> GodotVersion {
+        let version: Vec<&str> = godot4_prebuilt::GODOT_VERSION
+            .split('.')
+            .collect::<Vec<_>>();
+        GodotVersion {
+            full_string: godot4_prebuilt::GODOT_VERSION.into(),
+            major: version[0].parse().unwrap(),
+            minor: version[1].parse().unwrap(),
+            patch: version[2].parse().unwrap(),
+            status: "prebuilt".into(),
+            custom_rev: None,
+        }
```

Comment:
The "prebuilt" versions should always match "stable" releases, or is there a use case for treating the two differently?

If we ever support other prebuilt versions, like `4.1.rc0`, then we would lose that information by using "prebuilt" here...